Modules/Models/Configuration.swift

```diff
@@ -260,3 +276,35 @@ public extension Configuration {
     }
   }
 }
+
+public extension Dictionary where Key == String, Value == URL {
+  /// default values for FAQs
+  static var defaultFAQURL: [String: URL] {
+    let values = UserLanguage.allCases.map { lang in
+      // swiftlint:disable:next force_unwrapping
+      (lang.rawValue, URL(string: "https://get.immuni.gov.it/docs/faq-\(lang.rawValue).json")!)
+    }
+
+    return Dictionary(uniqueKeysWithValues: values)
+  }
+
+  /// default values for privacy notifice
+  static var defaultPrivacyNoticeURL: [String: URL] {
+    let values = UserLanguage.allCases.map { lang in
+      // swiftlint:disable:next force_unwrapping
+      (lang.rawValue, URL(string: "https://get.immuni.gov.it/docs/app-pn-\(lang.rawValue).html")!)
+    }
+
+    return Dictionary(uniqueKeysWithValues: values)
+  }
+
+  /// default values for privacy notifice
+  static var defaultTermsOfUseURL: [String: URL] {
+    let values = UserLanguage.allCases.map { lang in
+      // swiftlint:disable:next force_unwrapping
+      (lang.rawValue, URL(string: "https://get.immuni.gov.it/docs/app-tou-\(lang.rawValue).html")!)
```

Comment:
<!--
  0 failure: 
  2 warning:  swiftlint (0.39.2..., swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: Modules/Models/Configuration.swift;
  Line: 305;
-->

- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)
- :warning: swiftlint (0.39.2): SwiftLint rule 'implicitly_unwrapped_optional' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)